### ISSUE TITLE
Large number of minimap buttons?

### ISSUE THREAD

**Issue by deikun** [DungFu/NauticusClassic#2]
Getting a lot of minimap buttons displayed even with minimap settings switched off.

![image](https://user-images.githubusercontent.com/20842964/75289699-7678a700-57e4-11ea-86a8-044c8a6151fc.png)
![image](https://user-images.githubusercontent.com/20842964/75289765-9f993780-57e4-11ea-96e5-c84ccc435e0e.png)

**Comment by deikun** [comment]
Seems to be a byproduct of my minimap button bar provided by ProjectAzilroka addon for ElvUI. Looks like the minimap button is just one icon that has multiple layered icons underneath it that it swaps between?

**Comment by DungFu** [comment]
I can fix this in the next version, but an easy fix specific to MBB is to edit MBB.lua and add "NauticusClassicMiniIcon" to the MBB_Ignore List

Example:
```
MBB_Ignore = {
    ...
    [44] = "NauticusClassicMiniIcon"
}
```

**Comment by DungFu** [comment]
Fixed in v1.0.5
